fastlane/lib/fastlane/plugins/templates/action.rb.erb

```diff
@@ -0,0 +1,35 @@
+module Fastlane
+  module Actions
+    class <%= plugin_name.capitalize %>Action < Action
+      def self.run(params)
+        UI.message("The <%= plugin_name %> plugin is working!")
+      end
+
+      def self.description
+        %q{<%= summary %>}
```

Comment:
Is there any issue with escaping the `"` character? I really don't like the %q as it adds quite some unnecessary complexity. Also with `%q{` we also have to escape `}`, right?